addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -158,7 +171,7 @@ var QuestionFormWidget = publicWidget.Widget.extend({
                     .removeClass('d-none')
                     .find('.o_wslides_js_quiz_validation_error_text')
                     .text(renderedQuestion.error);
-            } else if (options.update) {
+            } else if (options?.update) {
```

Comment:
Shouldn't we be more explicit and either make `options` optional using `(options = {})` or pass an empty object from `_validateAndCreateNewQuestion`'s call?